api/src/org/labkey/api/data/dialect/PostgreSql91Dialect.java

```diff
@@ -1879,10 +1879,12 @@ public void setAdminWarning(HtmlString warning)
     }
 
     @Override
-    public void addAdminWarningMessages(Warnings warnings)
+    public void addAdminWarningMessages(Warnings warnings, boolean showAllWarnings)
     {
         if (null != _adminWarning)
             warnings.add(_adminWarning);
+        else if (showAllWarnings) // PostgreSqlDialectFactory.getStandardWarningMessage() is not accessible from here, so hard-code a sample warning
+            warnings.add(HtmlString.of("LabKey Server has not been tested against version 22.7. PostgreSQL 15.x is the recommended version."));

```

Comment:
22.7? How old fashioned.
```suggestion
            warnings.add(HtmlString.of("LabKey Server has not been tested against this version. PostgreSQL 15.x is the recommended version."));

```

Reply:
That was PG 22.7, which is still a very long way off. But okay.

Reply:
Ah, right. I misread the message.